pkg/operation/botanist/component/etcd/bootstrap_test.go

```diff
@@ -153,6 +153,17 @@ rules:
   - update
   - patch
   - create
+- apiGroups:
+  - coordination.k8s.io
+  resourceNames:
+  - druid-leader-election
```

Comment:
Is this correct? `druid-leader-election` instead of `etcd-druid-leader-election`? :)

Reply:
I am not sure about that actually. @amshuman-kr tried this resource name to test my PR in druid and he was able to run it. But I have to check if any resource name exists by this name. Wl let you know and update the PR.

Reply:
Okay. I removed the resource name and update the PR. It was adding unnecessary complication.

Reply:
Seems to have been correct https://github.com/gardener/etcd-druid/blob/841f2d016848d24c71423088ba3503f34bb9a3fd/main.go#L58